packages/cubejs-client-core/index.d.ts

```diff
@@ -8,6 +8,11 @@
  */
 
 declare module '@cubejs-client/core' {
+  type GeneratedMeasure = import('./generated').Measure;
```

Comment:
I don't think that it's a good idea to place generated files in client-core.

Maybe we can use interface merging and dynamic loading for the file.

https://www.typescriptlang.org/docs/handbook/declaration-merging.html m?

Because this generation CLI can be used for schema validation & typescript support. Do you have any ideas?